app/views/world_location_news/index.html.erb

```diff
@@ -21,78 +21,103 @@
+        } %>
+        <section class="featured-news items-<%= @feature_list.current_feature_count %>" id="featured-documents govuk-!-margin-bottom-8">
+          <% if @feature_list.any_current_features? %>
+          <div class="govuk-grid-row govuk-!-padding-bottom-8">
+            <%= render partial: 'shared/featured',
+              collection: @feature_list.current_featured,
+              as: :feature
+            %>
+          </div>
+          <% end %>
+        </section>
+      </div>
+    </div>
 
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
         <%= render partial: 'shared/recently_updated',
-                  locals: { recently_updated: @recently_updated,
-                            atom_url: atom_feed_url_for(@world_location),
-                            extra_class: 'panel',
-                            see_all_link: latest_path(world_locations: [@world_location]),
-                            documents_source: Locale.current.english? ? 'rummager' : nil } %>
-      </section>
+          locals: {
+            recently_updated: @recently_updated,
+            atom_url: atom_feed_url_for(@world_location),
+            see_all_link: latest_path(world_locations: [@world_location]),
+            documents_source: Locale.current.english? ? 'rummager' : nil 
+        } %>
+      </div>
     </div>
-  </div>
 
-  <div class="block our-mission">
-    <div class="inner-block">
-      <section class="article-section">
-        <h1 class="keyline-header"><%= t('world_location.headings.mission') %></h1>
-        <p>
-          <span class="mission_statement"><%= govspeak_to_html(@world_location.mission_statement) %></span>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds world-locations__our-mission">
+        <hr class="govuk-section-break govuk-section-break--visible world-locations__section-break govuk-!-margin-top-9 govuk-!-margin-bottom-3">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: t('world_location.headings.mission'),
+          margin_bottom: 3,
+          font_size: "l",
+        } %>
+        <p class="govuk-body">
+          <%= govspeak_to_html(@world_location.mission_statement) %>
         </p>
-      </section>
+      </div>
     </div>
-  </div>
 
-  <% if (@non_statistics_publications + @announcements + @statistics_publications).any? %>
-    <div class="block documents-grid">
-      <div class="inner-block">
-        <h1 class="block-title"><%= t('world_location.headings.documents') %></h1>
+    <% if (@non_statistics_publications + @announcements + @statistics_publications).any? %>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">    
+        <hr class="govuk-section-break govuk-section-break--visible world-locations__section-break govuk-!-margin-top-9 govuk-!-margin-bottom-3">
```

Comment:
If these `hr`'s are using override classes then there's no value in the bespoke class (`world-locations__section-break`). I'd recommend committing to a single ideology here: either use the class for additional spacing or just use the overrides.

Reply:
So, in theory I do agree however 😁
The `<hr>` [exists within the Design System](https://design-system.service.gov.uk/styles/typography/#section-break)

However this thicker black bar doesn't exist anywhere (by _anywhere_ the gem or the Design system - ideally it should)

I thought a nice halfway was to use the Design System as much as possible _then_ layer on top bespoke styling for the thicker black bar (as it then gets all the benefits of any release)

However I do take your point, Whitehall uses this quite a lot it might be better to make this style more global within this repo - what do you think?

![image](https://user-images.githubusercontent.com/71266765/126312317-c41e73cf-9319-4840-a4c2-93697d1b2349.png)

Reply:
In that case could we use the convention that the rest of the page uses and use a border on top of the section instead of the `hr`? I feel uncomfortable about this bespoke flavour of the section-break hanging around in whitehall.

Reply:
>I feel uncomfortable about this bespoke flavour of the section-break hanging around in whitehall.

I know what you mean. I feel like my preference this instance is to keep the `<hr>` and adjust the Design so it's inline with the Design System (make it thinner on this page)[1]

Opening this can up a tiny bit, it's a real mix across the site. Eg if you look on the footer or other pages sometimes it's a border, other times it's a `<hr>` i'm not convinced there's a clear convention.

https://www.gov.uk/coronavirus < Mix
https://www.gov.uk/access-to-elected-office-fund < `border`
https://www.gov.uk/ < Footer: mix of `border` and `<hr>` 
https://www.gov.uk/government/consultations/taking-action-on-climate-risk-improving-governance-and-reporting-by-occupational-pension-schemes-response-and-consultation-on-regulations < `border`

> In that case could we use the convention that the rest of the page uses and use a border on top of the section
 
I'm not sure that's 100% right either as this _is_ a `<hr>` semantically.
> You can use the `govuk-section-break` classes on an `<hr>`  element to create a thematic break between sections of content. 

Will check quickly with Design and decide: - make thin = `hr`, 
Keep thick = use border.

---
[1]
![image](https://user-images.githubusercontent.com/71266765/126486083-8f968980-cd8b-409e-82e1-bf650224751c.png)

Reply:
So @owenatgov this has been resolved in a better way - i've realised that [headings](https://components.publishing.service.gov.uk/component-guide/heading) on the gem have a border property. 🙌🏼